routes/web.php

```diff
@@ -94,8 +94,9 @@
     Route::post('/internet/internet_accesses/{id}/edit', 'InternetController@editInternetAccess')->name('internet.internet_accesses.edit');
 
     /** WIP: Admin Checkout **/
-    //Route::get('/admin/checkout', 'InternetController@showCheckout')->name('admin.checkout');
-
+    Route::get('/admin/checkout', 'Admin\AdminCheckoutController@showCheckout')->name('admin.checkout');
```

Comment:
For some reasons, I get 404 there. Not sure what could be the problem, probably on my side. Any idea? @kdmnk Already tried dumping autoloads...

Reply:
404 can mean any resource that is not found. Have you migrated the new PaymentType? (I haven't tested it either.)

Reply:
Gotcha!

Reply:
Thanks!